README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is *Kathleen Cannell*. I just finished my second year in the college.
+I am majoring in Economics:
+![Economics](https://i.kym-cdn.com/entries/icons/facebook/000/029/959/Screen_Shot_2019-06-05_at_1.26.32_PM.jpg) 
+
+I hope to minor in Inequality, Social Problems, and Change.
+
+**Background**
```

Comment:
You need at least 2 spaces for markdown to actually start a new line for you. Alternatively, you can use a small header here too (more #s)

Reply:
Thank you!